src/main/java/ch/njol/skript/expressions/ExprPermissions.java

```diff
@@ -43,37 +45,45 @@
 @Description("Returns all permissions of the defined player(s). Note that the modifications to resulting list do not actually change permissions.")
 @Examples("set {_permissions::*} to all permissions of the player")
 @Since("2.2-dev33")
-public class ExprPermissions extends PropertyExpression<Player, String> {
+public class ExprPermissions extends SimpleExpression<String> {
 	
 	static {
-		register(ExprPermissions.class, String.class, "permissions", "players");
+		Skript.registerExpression(ExprPermissions.class, String.class, ExpressionType.PROPERTY, "[(all [[of] the]|the)] permissions (from|of) %players%", "[(all [[of] the]|the)] %players%'s permissions");
```

Comment:
Does `[(all [[of] the]|the)] %players%'s permissions` make sense? At least **`the player's permissions`** or **`all player's permissions`** don't. Just "all [of] [the]"?

Reply:
Also the second patern should come first and the line should be split up to multi lines.

Reply:
Ugh right it confused me since the second pattern had "[(all [[of] the]|the)]" too..

The line is long enough and it doesn't look better than multi lines and having more lines doesn't change something.